Prep for 2.18.2 (#1668)
twilio/twilio-video.js: CHANGELOG.md

**Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.

2.18.2 (December 15, 2021)
==========================

Bug Fixes
---------

> - Fixed a bug where setting clientTrackSwitchOffControl to `auto` caused the RemoteVideoTrack's to get switched off even while playing in picture-in-picture window. (VIDEO-6677)
>   Note that this fix does not work on firefox because firefox does not yet implement [picture-in-picture](https://developer.mozilla.org/en-US/docs/Web/API/Picture-in-Picture_API) APIs.

Reviewer: ```suggestion
- Fixed a bug where setting `clientTrackSwitchOffControl` to `auto` caused the RemoteVideoTracks to get switched off while playing in picture-in-picture mode. Note that this fix does not apply to Firefox as it does not yet implement [picture-in-picture](https://developer.mozilla.org/en-US/docs/Web/API/Picture-in-Picture_API) APIs. (VIDEO-6677)
```
